datasource/virtual_machine/data.go

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type Config,Tag,DatasourceOutput
+package virtual_machine
+
+import (
+	"github.com/hashicorp/hcl/v2/hcldec"
+	"github.com/hashicorp/packer-plugin-sdk/common"
+	"github.com/hashicorp/packer-plugin-sdk/hcl2helper"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-sdk/template/config"
+	vsCommon "github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/common"
```

Comment:
Use `vsphere` to be consistent with the use seen in [post-processor/vsphere-template/post-processor.go](https://github.com/hashicorp/packer-plugin-vsphere/blob/32aceff8f8faeb9f45e50389fb21748f008292e5/post-processor/vsphere-template/post-processor.go#L22).

```suggestion
	vsphere "github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/common"
```